Sources/PointFree/Emails/SendEmail.swift

```diff
@@ -58,20 +58,16 @@ func sendEmail(
     )
 }
 
-// TODO: move to swift-prelude
-private func destructure<A, B, C, D>(
-  _ either: Either3<A, B, C>,
-  _ a2d: (A) -> D,
-  _ b2d: (B) -> D,
-  _ c2d: (C) -> D
-  )
-  -> D {
-    switch either {
-    case let .left(a):
-      return a2d(a)
-    case let .right(.left(b)):
-      return b2d(b)
-    case let .right(.right(.left(c))):
-      return c2d(c)
-    }
+func notifyError(subject: String) -> (Error) -> Prelude.Unit {
+  return { error in
+    _ = sendEmail(
+      to: adminEmails.map(EmailAddress.init(unwrap:)),
+      subject: "[PointFree Error] \(subject)",
+      content: inj1("\(error)")
+      )
+      .run
+      .perform()
```

Comment:
if you want this to be async you need to hit it with `parallel`. right now it blocks.